contracts/collect/StepwiseCollectModule.sol

```diff
@@ -0,0 +1,283 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {ICollectModule} from '@aave/lens-protocol/contracts/interfaces/ICollectModule.sol';
+import {Errors} from '@aave/lens-protocol/contracts/libraries/Errors.sol';
+import {FeeModuleBase} from '../FeeModuleBase.sol';
```

Comment:
Should we not rather import `FeeModuleBase` from the `lens-protocol` package? Doesn't look like there are any differences between the implementations

Reply:
Unfortunately there is a difference: I'm overloading the `_validateDataIsExpected` which is not `virtual` in the original `FeeModuleBase` from the `lens-protocol`, so I had to move it here and make it overloadable.